### PR TITLE
Missing EDS status update

### PR DESCRIPTION
**Pull request by otrosien**
The last scaling timestamp is not persisted on the EDS at the moment. Ensure that we call UpdateStatus.

### CODE REVIEW

**Comment by njuettner** [comment]
👍

**Comment by njuettner** [comment]
e2e tests fail due to:

```
3:35:28 PM autoscale_test.go:47
3:35:28 PM Error:      	Not equal:
3:35:28 PM expected: (*int32)(0xc00004c168)
3:35:28 PM actual  : (*int32)(0xc0005e06dc)
3:35:28 PM Test:       	TestEDSCPUAutoscaleUP
3:35:28 PM FAIL
```

File: cmd/e2e/basic_test.go [hunk 1]
```diff
@@ -72,7 +72,7 @@ func verifyEDS(t *testing.T, edsName string, edsSpec zv1.ElasticsearchDataSetSpe
 	eds, err := waitForEDS(t, edsName)
 	require.NoError(t, err)
 	err = waitForEDSCondition(t, eds.Name, func(eds *zv1.ElasticsearchDataSet) error {
-		if !assert.ObjectsAreEqualValues(replicas, eds.Spec.Replicas) {
+		if !assert.EqualValues(t, replicas, eds.Spec.Replicas) {
```
Comment (mikkeloscar) [inline cmd/e2e/basic_test.go]: I'm pretty sure they are timing out and #27 would fix it.

Comment (mikkeloscar) [reply cmd/e2e/basic_test.go]: But this change also looks fine actually.

Comment (otrosien) [reply cmd/e2e/basic_test.go]: I'm working on this. Should be easy.

**Comment by otrosien** [comment]
@mikkeloscar @njuettner this can finally be reviewed, I didn't add full coverage for es_client yet, just for the important parts. We can still add more later.

**Comment by mikkeloscar** [comment]
:+1:

**Comment by otrosien** [comment]
👍

**Pull request merged**